projects/templar/index.js

```diff
@@ -1,30 +1,24 @@
-const ADDRESSES = require('../helper/coreAssets.json')
-const { stakings } = require("../helper/staking");
-const { sumTokensExport, nullAddress, } = require("../helper/unwrapLPs");
+const { sumTokensExport } = require("../helper/unwrapLPs");
 
-const bscOwner = "0xEA724deA000b5e5206d28f4BC2dAD5f2FA1fe788";
 const bscStaking = "0xa1f61Ca61fe8655d2a204B518f6De964145a9324";
 const bscStakingV2 = "0xffC7B93b53BC5F4732b414295E989684702D0eb5";
-const bscTreasuryContract = "0xd01e8D805BB310F06411e70Fd50eB58cAe2B4C27";
 
 const bscTokens = {
-  BUSD: ADDRESSES.bsc.BUSD,
-  WBNB: ADDRESSES.bsc.WBNB,
-  MIM: "0xfe19f0b51438fd612f6fd59c1dbb3ea319f433ba",
-  TM: "0x194d1D62d8d798Fcc81A6435e6d13adF8bcC2966",
-  DAI: "0x1AF3F329e8BE154074D8769D1FFa4eE058B1DBc3",
   TEM: "0x19e6BfC1A6e4B042Fb20531244D47E252445df01",
-  VBUSD: "0x95c78222B3D6e262426483D42CfA53685A67Ab9D",
-  VBTC: "0x882C173bC7Ff3b7786CA16dfeD3DFFfb9Ee7847B",
 };
 
-const ethTokens = {
-  USDC: ADDRESSES.ethereum.USDC,
-  USDT: ADDRESSES.ethereum.USDT,
-};
-const ethOwner = "0x4Bd973e98585b003c31f4C8b9d6eAC5d3293B1e5"
-
 module.exports = {
+  bsc: {
+    tvl: sumTokensExport({
```

Comment:
the PR is mostly good, this is one part that I dont understand why is staking being counted as tvl?

Reply:
hmm, sorry but this is what we follow for all ohm forks, and protocols in general where tvl does not include staked coins. What if the dashboard is updated to use tvl info from treasury endpoint instead of tvl endpoint?

Reply:
Thanks for the suggestion, I will discuss it with my team to update our dashboard.